### DISCUSSION TITLE
In case someone face same problem - NativeWind cssInterop not mapping props correctly in 4.0.36

### DISCUSSION THREAD

**Discussion by spylinked** [nativewind/nativewind#1339] [General]
While using cssInterop in NativeWind to map custom props like classNameActiveColor, classNameInactiveColor, and tabClassName, the resolved styles were not being passed to the component props. This resulted in undefined values for props like activeColor and inactiveColor.

Here’s an example configuration that wasn’t working in earlier versions:
```

cssInterop(TestComponent, {
  classNameActiveColor: {
    target: false, // Prevent auto-mapping to style
    nativeStyleToProp: {
      color: "activeColor", // Map resolved `color` to `activeColor`
    },
  },
  classNameInactiveColor: {
    target: false, // Prevent auto-mapping to style
    nativeStyleToProp: {
      color: "inactiveColor", // Map resolved `color` to `inactiveColor`
    },
  },
  tabClassName: {
    target: "style", // Map directly to `style`
  },
});
```
Even when valid Tailwind classes were passed:

```
<TestComponent
  classNameActiveColor="text-red-500"
  classNameInactiveColor="text-gray-500"
  tabClassName="bg-blue-500"
/>
```
The console.log output was:

(NOBRIDGE) LOG  Resolved style: {"backgroundColor": "rgba(59, 130, 246, 1)"}
(NOBRIDGE) LOG  Resolved activeColor: undefined
(NOBRIDGE) LOG  Resolved inactiveColor: undefined

The issue was resolved after upgrading NativeWind to version 4.1.23. After the upgrade, the same cssInterop configuration worked as expected.

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
